charts/argo-cd/values.yaml

```diff
@@ -263,6 +263,12 @@ configs:
     # -- Enables use of the Progressive Syncs capability
     applicationsetcontroller.enable.progressive.syncs: false
 
+    # -- Enables [Applications in any namespace]
+    ## List of additional namespaces where applications may be created in and reconciled from.
+    ## The namespace where Argo CD is installed to will always be allowed.
+    ## Set comma-separated list. (e.g. app-team-one, app-team-two)
+    application.namespaces: ""
```

Comment:
It's from [upstream](https://github.com/argoproj/argo-cd/blob/master/docs/operator-manual/argocd-cmd-params-cm.yaml#L22-L27) 🙋 
doc: https://argo-cd.readthedocs.io/en/stable/operator-manual/app-any-namespace/#applications-in-any-namespace

<img width="712" alt="image" src="https://github.com/argoproj/argo-helm/assets/30188755/70023f29-0294-4868-80de-3f854ab1e016">